index.html

```diff
@@ -280,7 +284,44 @@ <h4><mark>Add forms</mark></h4>
         </p>
       </fieldset>
     </form>
+    <hr />
     <h2 id="scroll4">Lesson 4</h2>
-    <a href="./about/lesson4.html" target="_blank"> What do we offer</a>
+    <section class="advantage">
+      <h4 class="advantage__title">What do we offer</h4>
+      <ul class="advantage__list">
```

Comment:
I didn't see yellow line like in design

Reply:
<hr size="4" width="100" noshade color="#F5BD41" /> - we can add into hr tag in html file 
or
html
<hr />
css
hr {
  border: none;
  background: #F5BD41;
  overflow: visible;
  width: 100px;
  height: 4px;
}